clang/unittests/Format/ConfigParseTest.cpp

```diff
@@ -1441,8 +1441,9 @@ TEST(ConfigParseTest, GetStyleOfFile) {
   ASSERT_EQ(*Style9, SubSubStyle);
 
   // Test 9.8: use inheritance from a file without BasedOnStyle
-  ASSERT_TRUE(FS.addFile("/e/withoutbase/.clang-format", 0,
-                         llvm::MemoryBuffer::getMemBuffer("ColumnLimit: 123")));
+  ASSERT_TRUE(FS.addFile(
+      "/e/withoutbase/.clang-format", 0,
+      llvm::MemoryBuffer::getMemBuffer("BracedInitializerIndentWidth: 2")));
```

Comment:
I'd add it, not replace the old value.

Reply:
It doesn't matter which option to use as it's testing that an option from the parent config gets inherited, but a little redundancy wouldn't hurt here.